tools/docker/docker-compose-dev.yaml

```diff
@@ -71,13 +71,33 @@ services:
       postgres:
         condition: service_healthy
 
-  eda-worker:
+  eda-default-worker:
+    image: 'localhost/eda-default-worker'
+    build:
+      context: ../../
+      dockerfile: tools/docker/worker/Dockerfile
+    environment: *common-env
+    depends_on:
+      redis:
+        condition: service_healthy
+      postgres:
+        condition: service_healthy
+    volumes:
+      - '../../:/app/src:z'
+    restart: always
+
+  eda-activation-worker:
     deploy:
       replicas: ${EDA_NUM_WORKERS:-2}
-    image: 'localhost/aap-eda-worker'
+    image: 'localhost/eda-activation-worker'
```

Comment:
Here we don't need to build another tag, it can use the same image than the default worker, both can use the `localhost/aap-eda-worker`

Also I noticed that docker-compose-mac is not using this dockerfile but the regular image, we can do the same here, but in another PR. Take into account that this different worker image is to support the "local" deployment type, but now we have the podman type as default and we can use the same image for everything as we do in the docker-compose-mac.